portfolio-current/scripts/projects.js

```diff
@@ -1,23 +1,25 @@
 'use strict';
 
-var projects = [
-  {
-    name: "Project 1",
-    description: "Learning some new Awesome Platform"
-  },
-  {
-    name: "Project 2",
-    description: "Cool Algorithms"
-  },
-  {
-    name: "Project 3",
-    description: "Coding some amazing App"
-  }
-];
+var projects = [];
 
+$.getJSON("scripts/projects.json", function(result){
+            $.each(result, function(i, field){
+                projects.push(field);
+            });
```

Comment:
You do not need to do this since `result` is already an array. Thus you can just do `result.forEach` on line 10 instead of `projects.forEach`.